src/integration/pages/onDemandRadioEpisode/pashtoBrand/__snapshots__/canonical.test.js.snap

```diff
@@ -39,13 +39,31 @@ exports[`Canonical On Demand Radio Episode SEO Lang attribute 1`] = `"ps"`;
 exports[`Canonical On Demand Radio Episode SEO Linked data 1`] = `
 Object {
   "@context": "http://schema.org",
-  "@type": "AudioObject",
-  "description": "د نړۍ وروستي خبرونه",
-  "duration": "PT3M",
-  "embedURL": "https://polling.test.bbc.co.uk/ws/av-embeds/media/pashto/bbc_pashto_radio/w172xjcjkv7gw1w/ps?morph_env=live",
-  "name": "وروستي خبرونه",
-  "thumbnailUrl": "https://ichef.bbci.co.uk/images/ic/1024x576/p08b23c8.png",
-  "uploadDate": "2020-05-18T00:00:00.000Z",
+  "@type": "WebPage",
```

Comment:
because there 2 linked data scripts we are missing the audio object here

Reply:
Yes, I noticed. It actually does show up on the page though. I  do plan to refactor AudioPlayer to hoist the embedUrl logic needed by AudioObject. Then it should be easy to consolidate AudioObject into LinkedData. That will be my focus for today.

Reply:
Sounds good 😄  Is it ok to have it as part of this PR please? Thanks